packages/builder/src/components/nav/ModelSetupNav/ModelSetupNav.svelte

```diff
@@ -87,6 +87,10 @@
     notifier.success(
       "Success! Your changes have been saved. Please continue on with your greatness."
     )
+
+    setTimeout(() => {
+      $tourStore.tour.next()
```

Comment:
We don't need `setTimeout` here. Just call `$tourStore.tour.next()` once the model is saved above.

```js

 async function saveModel() {
    const errors = validate()
    if (errors.length > 0) {
      notifier.danger(errors.join("/n"))
      return
    }
    await backendUiStore.actions.models.save({
      model: $backendUiStore.draftModel,
    })
    notifier.success(
      "Success! Your changes have been saved. Please continue on with your greatness."
    )
     $tourStore.tour.next()  
}
```

Reply:
yea, I dont think its needed here. 
I had added the setTimeouts in most places because in some places the element for the next part of the tour wasn't visible in the DOM yet and it needed a small delay. This case should be ok without.